book/website/collaboration/inclusive-events/inclusive-events-tech.md

```diff
@@ -0,0 +1,219 @@
+
+Some hearing aids can also cause interference with older audio equipment resulting in shrill noises or "audio feedback" being heard. 
+This can be quite painful for many people to experience so it is important to check the compatibility of your devices and to arrange your event location in a way that minimizes the risk of audio issues.
+```
+
+In addition to the audio, ensure that your content and planning take into account the availability of screens and where they are located in the room. 
+This is especially important for in-person events that plan on using large venues or larger rooms.
+Screens and displayed content can be an important accessibility feature for attendees with sensory impairments who may rely on closed captions, sign language interpreters, or a zoomed-in view of any presented content due to visual impairments.
+
+Finally, for in-person or hybrid events, ensure that the sound system at your venue has an audio induction loop system that can send signals directly to hearing aids. 
+The availability of this system should be posted on a sign at the entrance to the room, along with specifications if only a certain part of the room is covered. 
+Consider additionally highlighting on your program or agenda what rooms and sessions at your event have support for audio induction loops, and who to contact to get access. 
+
+(cl-inclusive-events-technology-final-checks)=
+## What to check the day of the event
+As part of hosting, is it important to make time before the event starts to ensure that the technology supports you planned for are working and able to be implemented. 
+
+For example:
+* Check that any slides or presentations that will be shown during your event are legible and able to be displayed on your equipment. 
+* Check that any closed captioning software that you are using is functional and that the transcripts are able to be displayed and accessed by attendees.
+* Run through any slide or presentation changes to check that any video or slide transitions do not display bright flashes which present a health risk and can trigger photosensitive epilepsy
+* Check that audio induction loops are working and accessible to any attendees who need them.
+* Test the internet, cameras or recording equipment, and lighting. 
+* Ensure that wifi and cellular networks are accessible throughout the venue for any attendees who may need to check in with carers or other support staff who are not present at your event. 
+* Ensure that there is clear guidance that is readily accessible to attendees in the event of a technical issue. 
+
+
+Because this is not an exhaustive list, it is important to understand the needs and preferences of your specific attendees as part of making your event inclusive and accessible. 
+
+Where possible, involve a representative with lived experience in your event planning and work with a community to ensure that any barriers introduced by technology use at your event can be addressed and mitigated. 
```

Comment:
I guess a ref to 3307 is in order here? Or is that the one following this one? :)

Reply:
I think that's following. 

I added this suggestion after a reviewer pointed out that it needs to be stated everywhere possible that the event planning needs to involve people with the specific knowledge and needs that events are being organized to support. 

It'll be a repetitive sentiment that does bear repeating each time it's used.